src/components/Input/Input.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import Cleave from 'cleave.js/react';
+import * as InputMasks from './InputMasks';
+import './Input.scss';
+
+const Input = ({
+  autoComplete,
+  autoFocus,
+  className,
+  id,
+  inputMask,
+  isDisabled,
+  isRequired,
+  label,
+  name,
+  onBlur,
+  onChange,
+  onFocus,
+  placeholder,
+  type,
+  value,
+}) => {
+
+  const handleChange = (e) => {
+    onChange(e);
+  };
```

Comment:
does this need to be wrapped in another function?

Reply:
@ericyliu definitely doesn't need to be wrapped at the moment, however I've found these handlers if any refactoring is required in the future, E.G, adding any additional logic on an input change, that would happen in parallel to the callback. E.G: Applying a class, etc...

You are correct that it is useless at the moment, I just wrote it that way for some consistency across the component.

Reply:
thanks for the explanation! I always prefer to keep things as simple as possible, but I appreciate consistency :+1: